src/classes/ADDR_Seasonal_SCHED.cls

```diff
@@ -35,13 +35,34 @@
 */ 
 public class ADDR_Seasonal_SCHED implements Database.Batchable<SObject>, Schedulable {
 
-    private static final Integer BATCH_SIZE = 10;
+    @TestVisible
+    private final static Integer DEFAULT_BATCH_SIZE = 10;
+
+    @TestVisible
+    private Integer batchSize {
+        get {
+            if (batchSize == null) {
+                npo02__Households_Settings__c householdSettings = UTIL_CustomSettingsFacade.getHouseholdsSettings();
+
+                if (householdSettings.Seasonal_Addresses_Batch_Size__c == null
+                    || householdSettings.Seasonal_Addresses_Batch_Size__c < 0
```

Comment:
Let's check for 0 as well, ie `householdSettings.Seasonal_Addresses_Batch_Size__c <= 0`, thoughts?

Reply:
Good catch - turns if you use a batch size of 0, the batch just runs with the default 200. I just made the update to include 0